tasks/copy_to_glacier/handler.py

```diff
@@ -79,14 +80,28 @@ def task(event: Dict[str, Union[List[str], Dict]], context: object) -> Dict[str,
                     bucket (str)
             copied_to_glacier (list): List of S3 paths - one for each file copied
     """
+    #TODO: Possibly remove print statement and change to a logging statement.
     print(event)
     event_input = event['input']
     granules_list = event_input['granules']
     config = event['config']
 
     collection = config.get(CONFIG_COLLECTION_KEY)
     exclude_file_types = collection.get(COLLECTION_META_KEY, {}).get(EXCLUDE_FILE_TYPES_KEY, [])
-    glacier_bucket = config.get(CONFIG_BUCKETS_KEY).get('glacier').get('name')
```

Comment:
Might want to remove this key since we're going to be developing an override.

Reply:
Removed `CONFIG_BUCKETS_KEY`